QueueInteropSender.php

```diff
@@ -35,8 +35,8 @@ class QueueInteropSender implements SenderInterface
     public function __construct(
         EncoderInterface $messageEncoder,
         ContextManager $contextManager,
-        string $topicName,
```

Comment:
I'm not sure to get the value of changing the constructor's order... it's a BC break :)

Reply:
I know it is :disappointed: